docs/manual/programmability/eval-intro.md

```diff
@@ -438,3 +438,13 @@ it still has a different set of defaults compared to a script without a `#!` lin
 Another difference is that scripts without `#!` can run commands that access keys belonging to different cluster hash slots, but ones with `#!` inherit the default flags, so they cannot.
 
 Please refer to [Script flags](/topics/lua-api#script_flags) to learn about the various scripts and the defaults.
+
+## Read-only scripts
```

Comment:
i now realize this whole section is in `eval-intro.md`, doesn't it need to be in `_index.md` (of the `programmability` folder).
i.e. it's also about functions, not just EVAL  scripts.